### PR TITLE
Refactor native screens

### PR DESCRIPTION
**Pull request by im-samwong**
Related to task: https://app.clickup.com/t/86azt325t

In this pull request, I conducted a thorough refactoring and documentation update across all React Native screens in the application. The focus was on enhancing code readability and maintainability by adding descriptive comments, optimizing the use of React hooks, and ensuring consistent error handling and state management practices. This effort aimed to simplify future development and debugging, making the codebase more accessible to new and existing developers. The changes are aligned with best coding practices, contributing to the overall stability and scalability of the project.

### CODE REVIEW

**Review by Adissuu — APPROVED** [review]
Approved

File: apps/native/components/PropertyProfileComponent.tsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
-import React, { useEffect, useState } from "react";
-import UserPropertyForm from "./UserPropertyForm";
-import { db } from "../firebase";
-import { collection, addDoc } from "firebase/firestore";
+// PropertyProfileComponent.js
```
Comment (yasminehilout) [inline apps/native/components/PropertyProfileComponent.tsx:1]: Is that meant to be PropertyProfileComponent.tsx?

Comment (im-samwong) [reply apps/native/components/PropertyProfileComponent.tsx]: Yes, I have corrected it now.

**Review by Adissuu — APPROVED** [review]

**Review by Fetyali7 — APPROVED** [review]
Looks good to me!

**Review by yasminehilout — APPROVED** [review]
Good Work!

**Pull request merged**